iwp_website/controllers/loan_issue.py

```diff
@@ -17,23 +17,38 @@ class WebsiteLoanIssue(http.Controller):
 
     @http.route(
         [
-            '/loan/subscription',
+            '/struct/<int:struct_id>/loan/subscription',
+            '/struct/<int:struct_id>/loan/<int:loan_id>/subscription',
         ],
         type='http',
         auth='user',
         website=True,
     )
-    def subscribe_to_loan_issue(
-        self,
-        **post
-    ):
+    def subscribe_to_loan_issue(self, struct_id=None, loan_id=None, **post):
+        # self.reqargs contains request arguments but only if they pass
+        # checks.
+        self.reqargs = {}
+        # Get structure and perform access check
+        struct = request.env['res.partner'].sudo().browse(struct_id)
+        if not struct:
+            raise NotFound
```

Comment:
retourner une erreur plutot que lever une erreur?

Reply:
Le fait de lever l'erreur fait en sorte que le controlleur s'arrête et renvoit vers une page 404 not found par default.